lib/css.js

```diff
@@ -1,8 +1,7 @@
+import { warn } from './utils'
 const css = require('glamor')
 
-if (process.env.NODE_ENV !== 'production') {
-  console.error('Warning: \'next/css\' is deprecated. Please use styled-jsx syntax instead.')
-}
+warn('Warning: \'next/css\' is deprecated. Please use styled-jsx syntax instead.')
```

Comment:
once we have the wiki page up for glamor, we can link directly to it from here, instead of this half-baked suggestion